src/test/ui/issues/issue-44056.rs

```diff
@@ -10,6 +10,7 @@
 
 // compile-pass
```

Comment:
This test is `compile-pass`, why is it being run ? 

As in, we can cross-compile for `x86_64` with `avx` from everywhere AFAIK, and the tests checks that doing so works AFAICT, but this test shouldn't run.

Reply:
From the logs:

> ---- [ui] ui/run-pass/issues/issue-44056.rs stdout ----

it does appear that this test is in fact being run.

Reply:
See https://github.com/rust-lang/rust/pull/55667#issuecomment-435656808. This test was recently moved to compile-pass (but should not have been, as the original issue requires execution).

Reply:
Indeed, looking at the issue, this test does need execution. For that we would need a way to filter tests by target features available at run-time. Shouldn't be to hard to add a flag to prevent this tests from executing (e.g. using `is_x86_feature_detected!("avx")` ).